daemon/api.go

```diff
@@ -569,6 +570,31 @@ func getOpInfo(c *Command, r *http.Request) Response {
 	return SyncResponse(task.Map(route))
 }
 
+func postOp(c *Command, r *http.Request) Response {
+	route := c.d.router.Get(c.Path)
+	if route == nil {
+		return InternalError(nil, "router can't find route for operation")
+	}
+
+	id := muxVars(r)["uuid"]
+	task := c.d.GetTask(id)
+	if task == nil {
+		return NotFound
+	}
+
+	decoder := json.NewDecoder(r.Body)
+	var b bool
```

Comment:
There's a table in the rest docs for adding values. The type needed would depend on the operation, hopefully, and if/when we get that I expect the decoding of the value to get more complex. For now, just a boolean is enough, so why make it more complex? The external api wouldn't change when/if we need to make it richer (there's a table in rest.md for this).